backend/backendProcesses.cpp

```diff
@@ -104,7 +104,7 @@ void BackendProcesses::threadProcedure()
     all_bytes_in_minute.push_back("</bsr>");
 
     // only output the file when our buffer has reached 
-    if (all_bytes_in_minute.size() >= 10000 || min_time != last_minute) {
+    if (all_bytes_in_minute.size() >= 100000 || min_time != last_minute) {
```

Comment:
Noting this for later in case it becomes an issue, but I'll assume this was changed for a good reason for now